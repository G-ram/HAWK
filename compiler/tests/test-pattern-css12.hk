BEGIN{}
[@h2 *@]{
  print(this);
}
END{}
