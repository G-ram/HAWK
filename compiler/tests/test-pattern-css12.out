innerhtml=Attractiveness
nextChild=
class=mw-headline
id=Attractiveness
innerhtml=Mating Calls
nextChild=
class=mw-headline
id=Mating_Calls
innerhtml=Basketball Scouting Report
nextChild=
class=mw-headline
id=Basketball_Scouting_Report
innerhtml=Academic Success
nextChild=
class=mw-headline
id=Academic_Success
innerhtml=History
nextChild=
class=mw-headline
id=History
